gcc/rust/backend/rust-compile-type.cc

```diff
@@ -17,12 +17,14 @@
 // <http://www.gnu.org/licenses/>.
 
 #include "rust-compile-type.h"
+#include "rust-compile-context.h"
 #include "rust-compile-expr.h"
 #include "rust-constexpr.h"
 #include "rust-gcc.h"
 
 #include "tree.h"
 #include "stor-layout.h"
+#include <cstdio>
```

Comment:
we cant include cstdio it should be part of rust-system.h but I don't think your using any of the stuff from that header so you can remove this